apps/web/components/dialog/RescheduleDialog.tsx

```diff
@@ -77,7 +77,7 @@ export const RescheduleDialog = (props: IRescheduleDialog) => {
             />
 
             <DialogFooter>
-              <DialogClose>
+              <DialogClose asChild>
```

Comment:
There was an error in the console regarding <button> not being able to be under another element <button>, fixed.